src/scripts/Package-Toolkit.ps1

```diff
@@ -157,6 +157,15 @@ function Copy-TemplateFiles()
 
         Write-Verbose ("Compressing $srcPath to $zip" -replace (Get-Item $relDir).FullName, '.')
         Compress-Archive -Path "$srcPath/*" -DestinationPath $zip
+        
+        # Create unversioned copy for specific templates that need stable download links
+        if ($templateName -eq "finops-hub-copilot")
```

Comment:
We should not use one-off logic here. We should be consistent. Also this will break a test so that would also need to be updated.